### ISSUE TITLE
npz_file

### ISSUE THREAD

**Issue by zhongyuanxiang** [Justin900429/Facial-Graph-Representation-Learning#3]
Hello, how should I get this NPZ file

**Comment by Justin900429** [comment]
You can generate .npz with `compute_adj.py`.

```shell
$ python compute_adj.py \
    --npz_place <place to save the npz file for each LOSO> \
    --csv_name <csv file for the dataset>
```

**Comment by zhongyuanxiang** [comment]
I see. Thank you very much

**Comment by zhongyuanxiang** [comment]
I see. Thank you very much

**Comment by zhongyuanxiang** [comment]
您好，非常感谢您提供的代码，再一次打扰您。我有两个问题想请教一下您，1.我看了您更新的compute_adj.py为每个训练数据生成了一个npz文件，那么在train.py中加载文件的方式是否也应该修改呢？
`def LOSO_train(data: pd.DataFrame, sub_column: str, args,
               label_mapping: dict, device: torch.device):
    log_file = open("train.log", "w")

    npz_file = np.load(args.npz_file)`
2.我在使用dlib在SAMM数据集进行人脸检测时，部分图片出现无法检测到人脸的情况，例如020中的所有图片都无法检测到，我想请问一下，这是因为020中带了眼镜的原因吗？还是因为我的dlib版本问题呢？我的dlib版本是19.20，您的配置文件写的是19.22
再一次感谢您！

**Comment by Justin900429** [comment]
您好：

1. 生成的 npz file 是為每一個 LOSO data 都生成一個。
2. train 的部分我已經上傳最新版本的了，原本的是舊版的，謝謝您的提醒。
3. 我有在 README 更新 dlib weight 的下載方式了，您可以再試試看！

**Comment by zhongyuanxiang** [comment]
您好：非常感謝您的回復，我的dlib weight和您的是一樣的，還是無法檢測到人臉。我還有一個問題想請教一下您，代碼中網絡最後的輸出是5，但是label_mapping標簽還是8（以SAMM數據集爲例），并沒有做五分類處理，那最後在計算交叉熵損失的時候不就會出現錯誤嗎？
（train.py）
`loss = criterion(output, labels)`

**Comment by Justin900429** [comment]
您好

dlib 版本的差異應該不大，主要是看 weight file。另外 class 的部分是我先經過篩選的，paper 有提供哪些類別有留下，哪些需要去除。
